map/src/assets/main.css

```
@@ -0,0 +1,15 @@
+@import './base.css';
+@import 'node_modules/ol/ol.css';
```

```suggestion
@import '../node_modules/ol/ol.css';
```

This errors for me until I add the ../